config/jest/test.config.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
+    rootDir: "../../",
+    transform: {
+        "^.+\\.jsx?$": "<rootDir>/config/jest/test.transform.js",
+    },
+    testEnvironment: "jest-environment-jsdom",
+    globals: {
+        SNAPSHOT_INLINE_APHRODITE: true,
+    },
+    testMatch: ["<rootDir>/packages/**/*.test.js"],
```

Comment:
I find globs easier to grok than regexes for this kind of stuff.

Reply:
++

Reply:
Good catch.  I'll fix that.